data/planet-sources.yml

```
@@ -197,3 +197,21 @@
 - id: bbatsov
   name: Bozhidar Batsov
   url: https://batsov.com/feeds/OCaml.xml
+  only_ocaml: true
+- id: jonludlam
+  name: Jon Ludlam's Blog at recoil.org
+  url: "TODO: rss feed URL"
+- id: patricoferris
+  name: Patrick Ferris' Blog
+  url: "TODO: rss feed URL"
```

```suggestion
  url: https://jon.recoil.org/atom.xml
- id: patricoferris
  name: Patrick Ferris' Blog
  url: https://patrick.sirref.org/posts/atom.xml
```
(assuming @patricoferris wants that feed and not one of the others!)

---

Thanks David, confirming with Patrick.